bigtable-hbase-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/filters/PrefixFilterAdapter.java

```diff
@@ -53,4 +52,46 @@ public FilterSupportStatus isFilterSupported(
       FilterAdapterContext context, PrefixFilter filter) {
     return FilterSupportStatus.SUPPORTED;
   }
+
```

Comment:
Just thought that it made sense to keep prefix range calculations in PrefixFilterAdapter. But I don't have strong feelings...I moved it to its own utility